src/aboutwindow.cpp

```diff
@@ -13,21 +13,21 @@ AboutWindow::AboutWindow(QWidget *parent) : QWidget(parent), m_ui(new Ui::AboutW
     setWindowTitle(tr("About") + " " + qApp->applicationName());
```

Comment:
Please run `clang-format` on this file: `clang-format -style=file -i src/aboutwindow.cpp`.